__init__.py

```diff
@@ -991,15 +989,15 @@ def export_events(self, output_file, params, format='json', timezone_offset=None
         if self.timeout == 120:
             self.timeout = 1200
 
+        date_format = '%Y-%m-%d'
```

Comment:
does moving this out accomplish anything?

Reply:
yea it was inside of an if statement and then `date_format` was referenced later in another if statement that had the exact same condition so moved out of the  if statement